diff --git a/build.clj b/build.clj
index f021cfc..82f7f9b 100644
--- a/build.clj
+++ b/build.clj
@@ -71,22 +71,11 @@
 
 ;; ## Version and Releases
 
-(defn- version-info
-  "Compute the current version information."
-  ([opts]
-   (version-info opts false))
-  ([opts next?]
-   {:tag (str major-version
-              "."
-              (cond-> (parse-long (b/git-count-revs nil))
-                next? inc)
-              (when-let [qualifier (:qualifier opts)]
-                (str "-" qualifier))
-              (when (:snapshot opts)
-                "-SNAPSHOT"))
-    :commit (b/git-process {:git-args "rev-parse HEAD"})
-    :date (str (LocalDate/now))}))
 
+(def ^:private version-info
+  (constantly {:tag "@version@"
+               :commit ""
+               :date " "}))
 
 (defn print-version
   "Print the current version information."
diff --git a/deps.edn b/deps.edn
index aa3a00a..85cfb74 100644
--- a/deps.edn
+++ b/deps.edn
@@ -1,4 +1,7 @@
-{:paths ["resources" "src"]
+{:mvn/local-repo
+ "./.m2"
+
+ :paths ["resources" "src"]
 
  :deps
  {org.clojure/clojure {:mvn/version "1.11.1"}
